index.html

```diff
@@ -35,8 +35,9 @@ <h3>What is Open Source</h3>
 						<h3>Who's Open Source</h3>
 						<ul>
 							<li>Spotify, Netflix, Aws, Facebook, Twitter</li>
-							<li>Cooking receipts, templates job desc / code of conduct, legal docs etc</li>
-							<li>Government departments (open source by default)</li>
+							<li>Cooking receipts, templates job desc / code of conduct,  docs etc</li>
+							<li>Perhaps Natwest will appear in your list in 2 months time</li>
+							<li>Government departments (open source by default)</li>legal
```

Comment:
Thanks for the Pull Request.

`legal` needs to be between the opening `<li>` and closing `</li>